content/en/docs/reference/config/installation-options/index.md

```diff
@@ -24,10 +24,10 @@ $ istioctl manifest generate ... --set values.global.mtls.enabled=true
 Refer to [customizing the configuration](/docs/setup/install/istioctl/#customizing-the-configuration) for details.
 {{< /warning >}}
 
-{{< tip >}}
-Refer to [Installation Options Changes](/news/2019/announcing-1.3/helm-changes/)
-for a detailed summary of the option changes between release 1.2 and release 1.3.
-{{< /tip >}}
+{{< warning >}}
+This document is unfortunately out of date with the latest changes in the set of supported options.
+To get the exact set of supported options, please consult [`values.yaml`]({{< github_file >}}/install/kubernetes/helm/istio/values.yaml).
```

Comment:
each chart subdir has options as well.

e.g. 
/install/kubernetes/helm/istio/charts/galley/values.yaml

There are about 8 or 9 subcharts.

Reply:
OK, I've update this to point to the whole directory rather than a single file.